nirc_ehr/resources/web/nirc_ehr/buttons/clinicalObsGridButton.js

```diff
@@ -1,4 +1,4 @@
-EHR.DataEntryUtils.registerGridButton('ADDREQUIREDCLINICALOBS', function(config){
+EHR.DataEntryUtils.registerGridButton('NIRC_AUTO_POPULATE_DAILY_OBS', function(config){
     return Ext4.Object.merge({
         text: 'Add Required Clinical Obs',
```

Comment:
update text along since button name is updated

Reply:
done